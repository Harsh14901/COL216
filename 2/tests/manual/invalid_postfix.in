5*5112++5-1++789**+
14-+182*-3*19+*64--
396-3-+-20-0*33*1+*
89-73-+5-46**+37+7*
4842+-25**-7-9+*-37
*896-95**69*+5*90+*
18-2**08*3**65*+16+
5+-**95856*03-3-4+*
60*8+-6***764-*934+
*+6586+2+9*+81-8*+7
