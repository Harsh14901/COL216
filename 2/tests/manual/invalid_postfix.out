[*] Executing test case - 5*5112++5-1++789**+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 14-+182*-3*19+*64--
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 396-3-+-20-0*33*1+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 89-73-+5-46**+37+7*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 4842+-25**-7-9+*-37
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - *896-95**69*+5*90+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 18-2**08*3**65*+16+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 5+-**95856*03-3-4+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 60*8+-6***764-*934+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - *+6586+2+9*+81-8*+7
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
